src/node_sqlite.cc

```diff
@@ -289,6 +479,7 @@ void DatabaseSync::DeleteSessions() {
 DatabaseSync::~DatabaseSync() {
   if (IsOpen()) {
     FinalizeStatements();
+    FinalizeBackups();
```

Comment:
I can be wrong but I think you should put this outside this `if statement` since the backups can still be running while we close the database.

Reply:
Nice catch, @H4ad . TY.